src/main/java/make_comment/interface_adapter/makeCommentController.java

```diff
@@ -0,0 +1,23 @@
+package make_comment.interface_adapter;
+
+import make_comment.use_case.MakeCommentInputBoundary;
+import make_comment.use_case.MakeCommentRequestModel;
+
+public class makeCommentController {
```

Comment:
This goes for everything in the interface_adapter layer, but I think the classes should start with a capital "M"

Reply:
Just rename it through the refactor feature on intellij and it will update all the references for you!